Sources/Cache.swift

```diff
@@ -66,7 +66,7 @@ public final class Cache: Caching {
         #endif
     }
 
-    private static func defaultCostLimit() -> Int {
+    public static func defaultCostLimit() -> Int {
```

Comment:
As of Swift 4, they can’t be private because they’re used as default arguments in public methods (I believe constructors in this specific case).